### ISSUE TITLE
Implement Methods to Convert Between ZArrow and ZIO

### ISSUE THREAD

**Issue by adamgfraser** [zio/zio-arrow#5]
`ZArrow` is isomorphic to `ZIO` because of the `R` parameter in `ZIO`. We should add methods `toEffect` and `fromEffect` to express this and make it easy for users to move between the two datatypes.

```scala
trait ZArrow[+E, -A, +B] {
  def toEffect: ZIO[A, E, B]
}

object ZArrow {
  def fromEffect[E, A, B](zio: ZIO[A, E, B]): ZArrow[E, A, B]
}
```

Note that depending on what we do with #4 these type signatures may look even more natural.

**Comment by mtsokol** [comment]
Will try it if nobody has taken it

**Comment by adamgfraser** [comment]
Great!

**Comment by adamgfraser** [comment]
Resolved by #20.
